### PR TITLE
Update content_types.md

### PR DESCRIPTION
**Pull request by manez**
## Purpose / why

Another pass at applying some [Good Docs](https://thegooddocsproject.dev/) principles to our How-To pages, like in #1639.

## What changes were made?

Added an _Overview_ and _Before you start_ section to the opening, and made multiple small changes throughout for better [Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/) compliance.

## Verification

There are enough finicky bits that this needs an mkdocs build test to make sure all the parts work before merging, but anyone can review for content 😄 

But please do give me a minute to get the screenshot uploaded to this branch before review...

## Interested Parties

perhaps @rosiel or @ysuarez ?

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by manez** [comment]
Added the screenshot. Ready for review.

**Comment by manez** [comment]
This now has a PR from @rosiel in it, so another @Islandora/8-x-committers is needed for merging, if anyone has a minute to read some docs!

File: docs/user-documentation/content_types.md [hunk 1]
```diff
@@ -28,7 +30,7 @@ There are multiple tabs with different options to configure your Content Type:
 - _Manage form display_: Set the order in which fields appear in a form, including nesting; set how the user will enter data into a field (i.e., text field, drop-down list, radio butons, etc); set fields to be hidden in the display
```
Comment (dflitner) [inline docs/user-documentation/content_types.md]: There's a typo here "butons" and I suggest "set fields to be hidden in the display" be changed to "set fields to be hidden in the form."

File: docs/user-documentation/content_types.md [hunk 2]
```diff
@@ -28,7 +30,7 @@ There are multiple tabs with different options to configure your Content Type:
...
 - _Manage display_: Set how the data stored in the fields will be displayed on the Node. Custom display settings can be set for different "view vodes." For instance, a different view mode is applied for items using the Openseadragon viewer, which includes a field that displays the Media in Openseadragon instead of the standard Drupal image viewer.
```
Comment (dflitner) [inline docs/user-documentation/content_types.md]: Typo here: "view vodes"

**Review by dflitner — CHANGES_REQUESTED** [review]
I found a couple of typos that should be corrected but these updates are great.

**Comment by manez** [comment]
Thanks for catching those @dflitner ! View vodes, indeed.

All patched up now.

**Review by dflitner — APPROVED** [review]

**Pull request merged**